AcademyResidentInformationApi.Tests/V1/Gateways/AcademyGatewayTests.cs

```diff
@@ -280,6 +280,9 @@ private Person AddPersonRecordToDatabase(string firstname = null, string lastnam
             return databaseEntity;
         }
 
+        //Composite AcademyId key.
+        /*AcademyId is not a column in the DB but rather a string made up of the
+        claimId and personRef*/
         private static int ExtractClaimIdFromAcademyIdString(string academyId)
```

Comment:
I think you could remove this since we wouldn't need the Academy Id anymore.

Reply:
remove the comment?

Reply:
Sorry, I meant to remove the private method since we don't use `academyId` anymore in the production code.

I think it would cause some tests to need updating. 
But you should be able to replace usages of `academyId` with `claimId` and `personRef`

Reply:
This would break all the list endpoint tests. This isn't used in any of the get by id tests.

Reply:
That makes sense. Maybe it could be done separately. 
Agree that if it doesn't affect view endpoint tests, then removing it shouldn't be part of this PR.

Reply:
@LBHBomotoso and I have decided to have these changes that deal with the academy ID in a separate PR.

Reply:
@LBHBomotoso and I have decided to have these changes that deal with the academy ID in a separate PR.